homeassistant/components/radarr/sensor.py

```diff
@@ -82,6 +83,15 @@ class RadarrSensorEntityDescription(
         entity_registry_enabled_default=False,
         value_fn=lambda data, _: data,
     ),
+    "queue": RadarrSensorEntityDescription[int](
+        key="queue",
+        name="Queue",
```

Comment:
Let's move this to translations (as all other things moved already as well).

```suggestion
        translation_key="queue",
```

And add an entry to the `strings.json` file